pages_/my-shop.tsx

```diff
@@ -149,10 +116,11 @@ const MyShop = ({ isLoggedIn, shopId, encodedShopId }: Props) => {
   }, [isLoggedIn, shopId]);
 
   if (error) {
+    Sentry.captureException(error);
```

Comment:
we are catching global errors in the ErrorBoundary component and there we are using the captureException call

Reply:
gotcha, shall we remove all the occurrences of this?

Reply:
done